Tests/helper.py

```diff
@@ -191,9 +192,12 @@ def open_withImagemagick(self, f):
             raise OSError()
 
         outfile = self.tempfile("temp.png")
-        if command_succeeds([IMCONVERT, f, outfile]):
-            return Image.open(outfile)
-        raise OSError()
+        r = subprocess.run(
```

Comment:
Why change this from `call` to `run`? If all we're using is the `returncode`, then `call` would be simpler - https://docs.python.org/3/library/subprocess.html#subprocess.call

Reply:
The top of the linked Python docs say:

> The recommended approach to invoking subprocesses is to use the `run()` function for all use cases it can handle. For more advanced use cases, the underlying `Popen` interface can be used directly.

That is the only reason.

I agree that `.call()` is simpler, I'll change it.